internal/pkg/levee/testdata/src/example.com/tests/structlit/tests.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package structlit contains tests related to struct literals.
+package structlit
+
+import (
+	"example.com/core"
+)
+
+type Holder struct {
+	s core.Source
+	i core.Innocuous
+}
+
+type PointerHolder struct {
+	s *core.Source
+	i *core.Innocuous
+}
+
+func TestStructHoldingSourceAndInnocIsTainted(s core.Source, i core.Innocuous) {
+	h := Holder{
+		s,
+		i,
+	}
+	core.Sink(h) // TODO want "a source has reached a sink"
+}
+
```

Comment:
Consider adding the tests for an unnamed struct, e.g.
```golang
	h := struct{
		s *core.Source
		i *core.Innocuous
	}{
		s,
		i,
	}
	core.Sink(h) // TODO want "a source has reached a sink"
```

Reply:
I didn't notice until I saw the other PR, but this is an example of a `struct literal`.  I'm not sure what this file was meant to be testing given the name, since it does not test struct literals (beyond the underlying literals to the named type declarations).

Reply:
I think that is a very good test case to add! Done.

To my knowledge, all of them are struct literals: https://tour.golang.org/moretypes/5, so I think the name `structlit` is ok.

Reply:
I've been one step meta too long and was conflating the decl with the literal.

An int literal: `5`

A struct literal: `Foo{1, 2, 3}`.

`self.duh()`.